pegasus/sites.v3/code.org/views/volunteer_engineer_form.haml

```diff
@@ -128,7 +132,7 @@
     %button#btn-submit.btn.btn-default{type: "submit"}= I18n.t(:volunteer_engineer_submission_submit)
 
 #thanks{:style=>"display: none;"}
-  %p= I18n.t(:volunteer_engineer_submission_thankyou)
+  %p= I18n.t(:volunteer_engineer_submission_thankyou).gsub('%{url}', 'https://hourofcode.com')
```

Comment:
When we have the How-to Volunteer guide up, I'll change it to link directly to that.